### DISCUSSION TITLE
Bank of material parameters

### DISCUSSION THREAD

**Discussion by bibiboom89** [neperfepx/FEPX#108] [Ideas]
Hi everyone,

Happy new year to all of you.

I was wondering if there is a bank of material parameters somewhere that could be used as a starting point for simulations ?
If not, I think that could be a good idea to create one with the help of the community.
I'm looking for material parameters close to Ti64 and also any aluminum parameters fitted for cyclic behaviour. 

Thank you all,
Regards,
Bibi

**Comment by mkasemer** [comment]
@bibiboom89 : No, we don't have a library of material parameters currently. Generally, the easiest way to find material parameters is to scour trusted literature sources for matching (or near-match) material systems. Otherwise, you have to perform micromechanical experiments and optimize material parameters to these (it is generally not sufficient to fit to a macroscopic stress-strain curve, since many sets of parameters can give the same macroscopic response but very different grain-scale response.

For Ti64, you can find some material parameters here: https://doi.org/10.1016/j.actamat.2017.06.059

For aluminum, there is such a wide array of mechanical response based on the type of aluminum and processing history that you will have to consult literature to find the best match.

**Reply by bibiboom89** [reply]
Hi,

Thank you for your reply :)

Maybe a shared sheet could be posted here where the FEPX-Github community could share their material parameters on it with a reference to their article. On one side that gives visibility to the author, and on the other, FEPX users could find material parameters faster ?

The aluminum alloy in question is Al6082. 
I don't need exact material parameters, just something around would be enough, if you have some reference in mind ?

I got a bonus question concerning FEPX if you don't mind: assuming that my mesh has a large number of phases, all sharing the same material parameters except a few of them. Is there a way to define a default material to all the phases ? 

Thank you again for your support and your software FEPX,
Have a great day / weekend,
Bibi

**Reply by mkasemer** [reply]
Thanks again for your suggestion regarding a parameter dictionary, it is something we will look into in the future.

I suggest looking for articles where 6000-series aluminum alloys were modeled via CPFEM. This should get you in the ballpark. I have no specific suggestions.

Regarding your other question: this is currently not possible, and must be defined manually as you have suggested.
